focus_converter_base/pyproject.toml

```
@@ -1,8 +1,8 @@
 [tool.poetry]
 name = "focus-converter-base"
-version = "0.1.0"
+version = "1.0.0"
```

```suggestion
version = "0.7.0"
```

Pulling this back, we are not at full alignment with the FOCUS 1.0 release and even that is only at 1.0-preview.

---

ah, I am not sure how the release process works. I was assuming it would sit like this until we take an explicit action to release 1.0.0(.0)

---

generally yes, but as FOCUS 1.0-preview went out today I want to push what we have to main.